### PR TITLE
[Feature] Support Resnet strikes back

### PR DESCRIPTION
**Pull request by MengzhangLI**
## Motivation
Transfer learning of paper 'Resnet strikes back: An improved training procedure in timm'.

paper URL https://arxiv.org/pdf/2110.00476

Related PR: https://github.com/open-mmlab/mmdetection/pull/7001.

## Notification:
(1) Current MMSegmentation default ResNet backbone is ResNetV1c, which performs better than standard ResNet backbone. 
(2) This PR only implements on PSPNet.
(3) With more RSB (Resnet strikes back) experiments on different decoder heads (like DeepLabV3, FCN and so on) and RSB strategy on ResNetV1c backbone, we would consider open an independent `resnet_strikes_back` folder in `./configs/` like MMDetection.

## Results

<html>
<body>
<div class="okr-block-clipboard" data-okr="%7B%22okrDelta%22%3A%5B%7B%22lineType%22%3A%22unsupport%22%2C%22lineOptions%22%3A%7B%7D%2C%22lineContent%22%3A%5B%5D%7D%5D%2C%22businessKey%22%3A%22lark-doc%22%7D"></div><div data-zone-id="0" data-line-index="0" style="white-space: pre;">

Method | Backbone | optimizer | learning rate | weight decay | Detail config | Dataset | mIoU | mIoU (ms + flip)
-- | -- | -- | -- | -- | -- | -- | -- | --
PSPNet | ResNet RSB R50-D8 | AdamW | 0.0005 | 0.05 | KNET | Cityscapes | 78.47 | 79.45
PSPNet | ResNet V1c R50-D8 | SGD | 0.01 | 0.0005 | None | Cityscapes | **78.55** | **79.49**
PSPNet | ResNet R50-D8 | SGD | 0.01 | 0.0005 | None | Cityscapes | 78.22 | 79.46
  |   |   |   |   |   |   |   |  
PSPNet | ResNet RSB R50-D32 | AdamW | 0.0005 | 0.05 | KNET | Cityscapes | **74.09** | **77.18**
PSPNet | ResNet V1c R50-D32 | SGD | 0.01 | 0.0005 | None | Cityscapes | 73.88 | 76.85
PSPNet | ResNet R50-D32 | SGD | 0.01 | 0.0005 | None | Cityscapes | 72.61 | 75.51

</div>
</body>
</html>

Where `KNET` means learning policy referred from [K-Net](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/knet) semantic segmentation configs, which works well with AdamW optimizer:

```python
optimizer_config = dict(grad_clip=dict(max_norm=1, norm_type=2))
# learning policy
lr_config = dict(
    _delete_=True,
    policy='step',
    warmup='linear',
    warmup_iters=1000,
    warmup_ratio=0.001,
    step=[60000, 72000],
    by_epoch=False)
```

### CODE REVIEW

File: configs/pspnet/README.md [hunk 1]
```diff
@@ -55,6 +65,10 @@ Scene parsing is challenging for unrestricted open vocabulary and diverse scenes
...
+| PSPNet | R-50-D32 rsb   | 512x1024  |   80000 | 3.1 | 16.08              | 74.09 |         77.18 | [config](https://github.com/open-mmlab/mmsegmentation/blob/master/configs/pspnet/pspnet_r50-d32_rsb-pretrain_512x1024_adamw_80k_cityscapes.py)   | [model](https://download.openmmlab.com/mmsegmentation/v0.5/pspnet/pspnet_r50-d32_rsb-pretrain_512x1024_adamw_80k_cityscapes/pspnet_r50-d32_rsb-pretrain_512x1024_adamw_80k_cityscapes_20220316_141229-dd9c9610.pth) &#124; [log](https://download.openmmlab.com/mmsegmentation/v0.5/pspnet/pspnet_r50-d32_rsb-pretrain_512x1024_adamw_80k_cityscapes/pspnet_r50-d32_rsb-pretrain_512x1024_adamw_80k_cityscapes_20220316_141229.log.json)         |
```
Comment (MeowZheng) [inline configs/pspnet/README.md:69]: based on the foot note, R-50-D32 rsb is ResNetV1C arch?

Comment (MengzhangLI) [reply configs/pspnet/README.md:69]: It should be `R-50b-D32 rsb` rather than `R-50-D32 rsb`.

**Review by MeowZheng — APPROVED** [review]
